docs/primap-hist/index.html

```diff
@@ -412,6 +411,14 @@ <h3>Contact</h3>
 
   <h3>Changelog</h3>
 
+  <h4>Version 2.5.1 (February 2024)</h4>
+  <p>
+    The v2.5.1 release updates mainly third party data. New and updated input data sources are UNFCCC data for AnnexI, CDIAC fossil fuel related emissions (@hefnerGlobalRegionalNational2023a), EDGAR v8.0 data (CO$_2$, CH$_4$, N$_2$O), cement process emissions data (@andrewGlobalCO2Emissions2023), and data from India's third National Communication. A more detailed chnagelog is available with the zenodo release of the dataset (<a class="primap-blue" href="https://doi.org/10.5281/zenodo.10705513">https://doi.org/10.5281/zenodo.10705513</a>). Additionally we provide detailed per country changes <a class="primap-blue" href="changelog/html/index.html">here</a>.
```

Comment:
Is there another step with replacements for the citation identifiers? If not, needs to be replaced I guess.

Reply:
For this bit it's done in the update changelog step. The appearances of the current citation are updated automatically from the config.

Reply:
Neat!

Reply:
@JGuetschow Markdown `CO$2$` and citation (`@hefner ...`) identifiers are still there

Reply:
darn. When I copied over the text I thought that I need to fix that. I'll fix it now (after a meeting(. thanks for spotting.